torchdata/datapipes/iter/util/plain_text_reader.py

```diff
@@ -68,6 +70,16 @@ def return_path(self, stream: Iterator[D], *, path: str) -> Iterator[Union[D, Tu
         for data in stream:
             yield path, data
 
+    def as_tuple(self, stream: Iterator[List]) -> Iterator[Union[List, Tuple]]:
```

Comment:
I just realize we need to change the type hint here. The input `stream` might not be `iterator[List]`. We can use `Iterator[D] -> Iterator[Union[D, Tuple]]`.

Reply:
good point, done.